dts/bindings/mipi-dbi/zephyr,mipi-dbi-spi.yaml

```diff
@@ -31,5 +31,7 @@ properties:
   write-only:
     type: boolean
     description: |
-      Controller is not readable, IE only DOUT pin is connected on the SPI
-      interface.
+      Only writes to display controller, do not perform reads.
+      It allows to optimize performances.
```

Comment:
I might clarify here- if all controllers are `write-only`, then the read function implementation will be optimized out. This is more of a size optimization than a performance one